app/controllers/stories_controller.rb

```diff
@@ -86,9 +88,22 @@ def import
 
   def export
     csv = CSV.generate(headers: true) { |csv|
-      csv << CSV_HEADERS
       @project.stories.by_position.each do |story|
-        csv << story.attributes.slice(*CSV_HEADERS)
+        if params[:export_with_comments] == "1"
+          comments_headers = []
+          comments = {}
+          story.comments.each do |c|
+            comments["comment_#{c.id}"] = c.body
+            comments_headers.append("comment_#{c.id}")
+          end
+          csv << CSV_HEADERS + comments_headers
+          csv << story.attributes.merge(comments).slice(*CSV_HEADERS + comments_headers)
+        else
+          csv << CSV_HEADERS
+          @project.stories.by_position.each do |story|
```

Comment:
this is looping through the stories inside the other loop in line 90 when comments are not checked, this is rendering all stories multiple times

![image](https://github.com/fastruby/points/assets/188464/f2a33e91-276f-4650-85a9-f67096439214)

if we have 10 stories it will render the 10 stories 10 times